jenkinsfile-runner-steward-image/scripts/run.sh

```diff
@@ -23,12 +23,18 @@ declare -r PARAM_VARS_OPTIONAL=(
   'PIPELINE_LOG_FLUENTD_HOST'
   'PIPELINE_LOG_FLUENTD_PORT'
   'PIPELINE_LOG_FLUENTD_TAG'
+  'PIPELINE_CLONE_RETRY_INTERVAL_SEC'
+  'PIPELINE_CLONE_RETRY_TIMEOUT_SEC'
   'JOB_NAME'
   'RUN_NUMBER'
   'RUN_CAUSE'
   'TERMINATION_LOG_PATH'
 )
 
+declare -r \
+  DEFAULT_PIPELINE_CLONE_RETRY_INTERVAL_SEC=15 \
```

Comment:
Is there any reason behind taking exactly 15 sec?

Reply:
No actually I increased it as we decided to increase from 30 seconds timeout into 180 seconds. If you think we should have 5 seconds interval with 180 seconds timeout I'm fully fine with it

Reply:
I think it is ok. It has pros and cons. Let's keep 15 sec.

Reply:
Anyhow we made it configurable in a way these values can be changed in stewardci-core or in backend systems later on

Reply:
https://github.com/SAP/stewardci-core/pull/209